src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/DatabaseTreeNode.cs

```diff
@@ -79,8 +79,10 @@ public bool IsAccessible(SmoQueryContext context)
             }
             catch (Exception ex)
             {
-                // IsAccessible is not set of DW Gen3 so exception is expected in this case
-                if (IsDWGen3(context?.Database))
+                // IsAccessible is not set of DW Gen3 and Dataverse so exception is expected in this case
+                // Incase of dataverses, isAccessible creates a temp table to check if the database is accessible, however dataverse 
+                // don't support ddl statements and therefore this check fails.
+                if (IsDWGen3(context?.Database) || IsDataverse(context?.Database))
```

Comment:
@Charles-Gagnon and @shueybubbles if I change this check to unknown, won't it make else branch useless?